### PR TITLE
44 login module

### LINKED ISSUE #44: Login module

### PR DESCRIPTION
**Pull request by SakshiShreya**

### CODE REVIEW

File: src/common/auth/signup.tsx [hunk 1]
```diff
@@ -80,7 +90,6 @@ export default function Signup({ onLogin, onClose }: Props) {
           onError={() => {
             console.log("Login Failed");
           }}
-          useOneTap
```
Comment (Soumya-Shreya) [inline src/common/auth/signup.tsx:83]: Why is this removed?

**Review by Soumya-Shreya — APPROVED** [review]

File: src/common/auth/signup.tsx (hunk 1 above)
Comment (SakshiShreya) [reply src/common/auth/signup.tsx:83]: Because useOneTap will show login with google popup on top left.
But right now that popup is coming when we open login/signup button. But that should ideally open when we open the site for the first time.
